frontend/src/pages/PhishingEmailInfo/index.js

```diff
@@ -112,16 +112,24 @@ function PhishingEmailInfo({
           {isShowing && phishingEmails[page].participantActions.length > 0 && (
             <EmailItemField>
               {phishingEmails &&
-                phishingEmails[page].participantActions.map(info => (
-                  <EmailItemContent key={info.actionType}>
+                phishingEmails[page].participantActions.map((info, i) => (
+                  <EmailItemContent
+                    fault={
+                      info.actionId ===
+                        phishingEmails[page].participantBehaviour.actionId &&
+                      'fault'
+                    }
+                    key={info.actionType + i}
```

Comment:
I think we should use actionId here instead of using the index
```suggestion
                    key={info.actionId}
```